app/_hub/kong-inc/opa/index.md

```diff
@@ -0,0 +1,307 @@
+        "port": "8000",        # port to which the request was made
+        "tls": {},             # TLS details if the request was made on HTTPS and Kong terminated the TLS connection
+        "method": "GET",       # HTTP method used in the request
+        "scheme": "http",      # protocol used to make the request by the client, this can be either `http` or `https`
+        "querystring": {       # Query string in the HTTP request as key-value pairs
+          "foo" : "bar",
+          "foo2" : "bar2",
+        },
+        "headers": {           # HTTP headers of the request
+          "accept-encoding": "gzip, deflate",
+          "connection": "keep-alive",
+          "accept": "*\\/*"
+        }
+      }
+    },
+    "client_ip": "127.0.0.1",# client IP address as interpreted by Kong
+    "service": {             # The Kong service resource to which this request will be forwarded to if OPA allows. Injected only if `include_service_in_opa_input` is set to `true`.
+      "host": "httpbin.org",
+      "created_at": 1612819937,
+      "connect_timeout": 60000,
+      "id": "e6fd8b19-89e5-44e6-8a2a-79e8bf3c31a5",
+      "protocol": "http",
+      "name": "foo",
+      "read_timeout": 60000,
+      "port": 80,
+      "updated_at": 1612819937,
+      "ws_id": "d6020dc4-67f5-4c62-8b45-e2f497c20f5c",
+      "retries": 5,
+      "write_timeout": 60000
+    },
+    "route": {               # The Kong route that was matched for this request. Injected only if `include_route_in_opa_input` is set to `true`.
+      "id": "bc6d8617-76a7-441f-aa40-32eb1f5be9e6",
+      "paths": [
+        "\\/"
+      ],
+      "protocols": [
+        "http",
+        "https"
+      ],
+      "strip_path": true,
+      "created_at": 1612819949,
+      "ws_id": "d6020dc4-67f5-4c62-8b45-e2f497c20f5c",
+      "request_buffering": true,
+      "updated_at": 1612819949,
+      "preserve_host": false,
+      "regex_priority": 0,
+      "response_buffering": true,
+      "https_redirect_status_code": 426,
+      "path_handling": "v0",
+      "service": {
+        "id": "e6fd8b19-89e5-44e6-8a2a-79e8bf3c31a5"
+      }
+    },
+    "consumer": {            # Kong consumer that was used for authentication for this request. Injected only if `include_consumer_in_opa_input` is set to `true`.
+      "id": "bc6d8617-76a7-431f-aa40-32eb1f5be7e6",
+      "username": "kong-consumer-username"
+    }
+  }
+}
+
+```
+
+> Please note that it is possible that Consumer and Service resources are not present for any given request in Kong.
+
+## Expected response from OPA
+
+Once OPA is done executing policies, the plugin expects the policy evaluation
+result in one of the two defined formats. Any other format will result in the
+plugin returning a `500 Internal Server Error` to the client.
+If the HTTP status code of the response from OPA is not `200 OK`, then it is
+treated as an error, which results in a `500 Internal Server Error` to the client.
```

Comment:
Starting from "Any other format" and to the end of the paragraph, this section repeats itself. If what this is trying to say is: correct format matching one of two options = `200 OK`, wrong format = `500 Internal Server Error`, then there must be a more concise way to say it.

Reply:
> Starting from "Any other format" and to the end of the paragraph, this section repeats itself. If what this is trying to say is: correct format matching one of two options = `200 OK`, wrong format = `500 Internal Server Error`, then there must be a more concise way to say it.

@lena-larionova made an attempt in commit e7fb114. You and @hbagdi please review.

Reply:
Looks great, thanks!